django/thunderstore/community/api/experimental/views.py

```diff
@@ -69,6 +79,61 @@ def get_queryset(self):
         return community.package_categories
 
 
+class PackageValidationExperimentalApiView(APIView):
+    @swagger_auto_schema(
+        manual_parameters=[
+            Parameter(
+                "mods",
+                IN_QUERY,
+                "Canonical package names including version numbers",
+                type=TYPE_ARRAY,
+                items=Items(type=TYPE_STRING),
+            ),
+        ],
+        responses={
+            200: PackageValidationResponseSerializer(),
+            400: "Unknown community_id in path",
+        },
+        operation_id="experimental.community.package_validation",
+        operation_description="Validate list of packages exist in a community",
+    )
+    def get(self, request: Request, community_id: str):
+        try:
+            community = Community.objects.listed().get(identifier=community_id)
+        except Community.DoesNotExist:
+            return Response(f'Unknown community "{community_id}"', HTTP_400_BAD_REQUEST)
+
+        submitted_mods = [m.strip() for m in request.GET.getlist("mods")]
+        errors: List[str] = []
+
+        if not submitted_mods:
+            return Response({"validation_errors": errors})
+
+        listings: QuerySet[PackageListing] = (
+            community.package_listings.filter(package__is_active=True)
+            .filter(package__is_deprecated=False)
+            .filter(package__owner__is_active=True)
```

Comment:
Packages can be active even if their owners aren't IIRC, was this copied from somewhere that also did it this way? If yes then no need to change this, but we should probably unify the visibility rules to one place

Reply:
This isn't copy-paste, I just quickly checked which models have `is_active` etc. and threw it into a filter. I guess it makes sense that the package can live on even if the team is disbanded, so I'll remove `package__owner__is_active` check.

Also replaced `package__is_active` & `package__versions__is_active` checks with existing QuerySet method `active()`.

Does the check for package deprecation make sense in this context?

Reply:
It doesn't no, deprecated packages should be selectable too

Reply:
Removed